src/index.ts

```diff
@@ -62,14 +62,15 @@ const MAX_RETRYCOUNT = 36;
 let cachedAuthResponse: any;
 let answers: Answers = {};
 
-const program = new Command(packageJson.name)
+const program = new Command('pcs')
     .version(packageJson.version, '-v, --version')
     .option('-t, --type <type>', 'Solution Type: remotemonitoring', /^(remotemonitoring|test)$/i, 'remotemonitoring')
     .option('-s, --sku <sku>', 'SKU Type: basic, standard, or test', /^(basic|standard|test)$/i, 'basic')
     .option('-e, --environment <environment>',
             'Azure environments: AzureCloud or AzureChinaCloud',
             /^(AzureCloud|AzureChinaCloud)$/i, 'AzureCloud')
     .option('-r, --runtime <runtime>', 'Microservices runtime: dotnet or java', /^(dotnet|java)$/i, 'dotnet')
+    .option('--release <release>', 'Microservices release version: currently published or testing', /^[\w\.\-\_]{1,64}$/i, 'testing')
```

Comment:
I don't think we should allow the user to enter an arbitrary string - I think we need to offer a selection of known releases that and ask the user to select amongst them.  I also think we should consider whether to lookup the appropriate tags underneath that release per container or whether the tags will always be consistent.

Reply:
If we are just talking about versions that we released sure it makes sense to show a list of releases, but if there are releases that are outside of what we pushed to docker we can't possibly know their values. Anyways lets chat offline.